### PR TITLE
feat(i18n): add german translations

### PR DESCRIPTION
**Pull request by brodi1**
- added GenericName and Comment

### CODE REVIEW

File: src/po/swappy.desktop.in [hunk 1]
```diff
@@ -3,9 +3,11 @@ Name=Swappy
 GenericName=Annotation Tool
 GenericName[fr]=Outil d'annotation
 GenericName[pt_BR]=Ferramenta de Anotação
+GenericName[de]=Anmerkungswerkzeug
```
Comment (jtheoof) [inline src/po/swappy.desktop.in:6]: minor do you mind putting it in alphabetical order ? (before the `fr` one)

File: src/po/swappy.desktop.in [hunk 2]
```diff
@@ -3,9 +3,11 @@ Name=Swappy
...
 Comment=A Wayland native snapshot editing tool
 Comment[fr]=Un outil d'édition de capture d'écran avec support natif pour Wayland
 Comment[pt_BR]=Uma ferramenta de edição de snapshot nativa do Wayland
+Comment[de]=Ein natives Wayland Bildschirmfoto-Bearbeitungswerkzeug
```
Comment (jtheoof) [inline src/po/swappy.desktop.in:10]: same thing

File: src/po/swappy.desktop.in (hunk 1 above)
Comment (brodi1) [reply src/po/swappy.desktop.in]: @jtheoof done. see https://github.com/jtheoof/swappy/pull/95/commits/01639cf2f7e7f7ee95963332b29116ad927bb297
for some reason it didn't pass the first check?! 😕

**Comment by jtheoof** [comment]
Can you squash both commits and make sure the linter passes ? Thanks

File: src/po/swappy.desktop.in (hunk 1 above)
Comment (jtheoof) [reply src/po/swappy.desktop.in]: Your commit is not respecting the guidelines in the README. I suggest to squash both commits and use: `fix(i18n): add german translations to desktop file`

Comment (jtheoof) [reply src/po/swappy.desktop.in]: Never mind, I could do it from the Github UI. Thanks for the update.

**Pull request merged**